src/content/blog/astro-580.mdx

```diff
@@ -0,0 +1,77 @@
+
+```sh
+node -v
+```
+
+You will need to update both your local development environment and any CI/CD environments to use Node.js 18.20.8 or later. You can check the docs for your [deployment platform](https://docs.astro.build/en/guides/deploy/) to see how to set the Node.js version. This might involve choosing an option in a dashboard, setting an environment variable, or adding a configuration file to your project. Many platforms will respect the `NODE_VERSION` environment variable or a [`.nvmrc`](https://github.com/nvm-sh/nvm#nvmrc) file. For example, to set the Node.js version to 22, you can create a `.nvmrc` file in your project root with the following content:
+
+```ini title=".nvmrc"
+22
+```
+
+See the [nvm docs](https://github.com/nvm-sh/nvm#nvmrc) for more details on how to use `.nvmrc` files.
+
+<Note title="Using Cloudflare Pages?">
+The current build image for Cloudflare Pages uses Node.js 18.17.1 by default, which is no longer supported by Astro. If you are using Cloudflare Pages you should [override the default Node.js version](https://developers.cloudflare.com/pages/configuration/build-image/#override-default-versions) to Node.js 22. This does not affect users of Cloudflare Workers, which uses Node.js 22 by default.
+</Note>
+
+For more information on Astro's Node.js support policy, see our [upgrade guide](https://docs.astro.build/en/upgrade-astro/#nodejs-support-and-upgrade-policies).
+
+## Bug fixes
+
+As always, we've been working hard on fixing issues since the [5.7 release](/blog/astro-570). See [the changelog](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md) for all the details.
+
+## Community
+
+The Astro core team is:
+
+<Mention name="ben" />, <Mention name="caleb" />, <Mention name="chris" />, <Mention name="ema" />, <Mention name="erika" />, <Mention name="florian" />, <Mention name="fuzzy" />, <Mention name="hideoo" />, <Mention name="luiz" />, <Mention name="matt" />, <Mention name="matthew" />, <Mention name="nate" />, <Mention name="reuben" />, <Mention name="sarah" />, and <Mention name="yan" />.
+
+Thanks to all the other contributors who helped make Astro 5.8 possible, including: [Adam Matthiesen](https://github.com/Adammatthiesen), [Alan](https://github.com/altano), [Alexey Zimarev](https://github.com/alexeyzimarev), [Angelo Verlain Shema](https://github.com/vixalien), [Armand Philippot](https://github.com/ArmandPhilippot), [Junseong Park](https://github.com/jsparkdev), [Katsuyuki Karasawa](https://github.com/L4Ph), [knj](https://github.com/jp-knj), [Leo Developer](https://github.com/Le0Developer), [Lofty-Brambles](https://github.com/Lofty-Brambles), [Luiz Gonzaga dos Santos Filho](https://github.com/lfilho) [Martin Trapp](https://github.com/martrapp), [Matt Yoho](https://github.com/mattyoho), [Ming-jun Lu](https://github.com/mingjunlu), [Petr Paločko](https://github.com/Fakerko), [Tanishq Manuja](https://github.com/tanishqmanuja), and [翠](https://github.com/sapphi-red).
```

Comment:
Will note that we *could* be explicit about our community acknowlegements, if we wanted:

> Thanks to all the other contributors who helped make Astro 5.8 possible, with code and docs additions and improvements since the last minor release, including: 

(I just like our "everything that happened since the last minor release to get us to where we are now" framing)